binsplit.go

```diff
@@ -12,8 +12,8 @@ import (
 
 var Debug bool = false
 
-const ApplicationDescription = "Binary File Splitting"
-const BuildVersion = "dev"
+const applicationDescription = "Binary File Splitting"
+const buildVersion = "dev-2"
```

Comment:
this should have been left as `dev`.
please update `.version` file in the root directory.

ps: I need to write some guidelines on that.